.github/workflows/build.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ["3.10", "3.9", "3.8", "3.7"]
+        python-version: ["3.10", "3.9", "3.8"]
```

Comment:
Maybe this should be an other PR, but I wanted to use `:=` in my code... let me know if I should split this one in 2.